CommonApplication.cmake

```diff
@@ -70,6 +70,11 @@ if(APPLE)
     WORKING_DIRECTORY ${_INSTALLDIR})" COMPONENT apps)
 elseif(WIN32)
```

Comment:
MSVC. WIN32 means also cygwin, where this is probably not needed as it behaves like Linux.